### ISSUE TITLE
Add support for higher-order central moments (for skewness/kurtosis)

### ISSUE THREAD

**Issue by Balandat** [prestodb/presto#7481]
Presto today supports computing variance and covariance. It would be great to add support for computing higher-order central moments (third and fourth order). 

The way to do this in a single pass is known, see e.g. [this paper](http://prod.sandia.gov/techlib/access-control.cgi/2008/086212.pdf) for the formulas. This would be a relatively simple addition that would allow for additional meaningful data analysis. 

A sample implementation is [here](https://www.johndcook.com/blog/skewness_kurtosis/)

**Comment by kaschaeffer** [comment]
@erichwang @AlekseiS any objections to adding this to Presto? this would be helpful for some of the more advanced stats that core data science team is exploring in collaboration with Deltoid.

**Comment by AlekseiS** [comment]
For global functions let's ask @martint .

**Comment by Balandat** [comment]
So I went ahead implemented this. I have it running successfully. Basically, there is a new "CentralMomentsState" and associated update/merge functions in AggregationUtils that does a bunch more than than the existing "VarianceState". For now I implemented the following functions: 
```
skewness(x) -> double
kurtosis(x) -> double 
```
both of which work just like the variance aggregation function. 
I'm also considering implementing a function 
```
central_moment(x, k) -> double
```
that computes the non-standardized central moment of order k, which wouldn't require anything to the state update / merge logic. 

I didn't touch any of the existing code. If this can go into the global functions as planned I'm happy to start writing some tests and documentation and submit a pull request.

**Comment by martint** [comment]
Yeah, those functions are good. Let's add them.

**Comment by Balandat** [comment]
This builds and tests fine locally. It appears the test error in travis is due to #7512.

**Comment by nezihyigitbasi** [comment]
Fixed thru #7504
